blast/blast_data_gen

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/perl
+
+$num_args = $#ARGV + 1;
+if ($num_args != 2) {
+    print "Usage: blast_data_gen number_contigs length_of_contigs\n";
```

Comment:
I currently do not seed the random number generator but can if we would like predictable results.